docs/designs/2686-proposal-process.md

```diff
@@ -0,0 +1,90 @@
+# Proposing Changes to Tanzu Community Edition
+
+* Proposal: [https://github.com/vmware-tanzu/community-edition/issues/2686](https://github.com/vmware-tanzu/community-edition/issues/2686)
+
+## Introduction
+
+Tanzu Community Edition relies on design proposals to assess the viability of non-trivial changes
+to the project. This document covers details on the proposal process, which can also be used as
+guidelines for how to create a proposal.
+
+## The Proposal Process
+
+The proposal process is the process for reviewing a proposal and reaching a decision about whether
+to accept or decline the proposal.
+
+1. The proposal author [creates a brief issue](https://golang.org/issue/new) describing the proposal.
```

Comment:
Should we get an issue template added specifically for this?

Reply:
As talked about in review today, we're going to try for a while without a specific issue template and see if converting existing issues (e.g. `kind/feature`) to a proposal is adequate.